src/components/Footer/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FC } from "react";
+
+import Logo from "../Logo";
+import P from "../P";
+import Icon from "../Icon";
+import Row from "../Row";
+import Column from "../Column";
+import Mail from "../../assets/img/icons/mail.svg";
+import {
+  FooterWrapper,
+  Wrapper,
+  ContactWrapper,
+  FooterP,
+  MailA,
+} from "./styledComponents";
+import { white, copyrights } from "../../assets/styles/colors";
+
+const Footer: FC = () => {
+  return (
+    <FooterWrapper>
+      <Wrapper>
+        <Logo isBackgroundDark width="137px" />
+        <ContactWrapper>
+          <FooterP>Kontakt</FooterP>
+          <Row>
+            <Icon color={white} width="1em">
+              <img src={Mail} />
+            </Icon>
+            <MailA href="mailto:myslaceklasy@email.com">
+              myslaceklasy@email.com
+            </MailA>
+          </Row>
+        </ContactWrapper>
+      </Wrapper>
+      <Column>
+        <P color={copyrights} textAlign="center" fontSize="0.875em">
+          © Copyrights 2020.
```

Comment:
I think there is 4 px bottom padding missing here, but I think line-height is bigger 28px instead of 24px I think 😀

Reply:
I added the padding-bottom but I left line-height as it was - 24px. I think line-height here depends on where you check in on the design ;) In this view on every device is 24px: https://www.figma.com/file/FZKtRjUFs3QHXZgQKnJRMW/Thinking-Classroom?node-id=474%3A187

Reply:
Oh no, misunderstanding :D I meant it should be 24 px, but I thought it was 28px in your implementation, nvm, all fine now.